ui/src/containers/BeamlineActionsContainer.jsx

```diff
@@ -76,124 +72,152 @@ class BeamlineActionsContainer extends React.Component {
     const defaultDialogPosition = { x: -100, y: 100 };
 
     return (
-      <Row>
-        <Col xs={12}>
```

Comment:
Unrelated: I'm removing an unnecessary row/col wrapping in this component. The rest is Prettier formatting (ignore whitespace changes for a cleaner diff).